README.md

```diff
@@ -114,6 +114,9 @@ We also have explicit _non-goals_ for Carbon, notably including:
 
 -   A stable ABI for the entire language and library
 -   Perfect backwards or forwards compatibility
+-   Support legacy code for which source code is unavailable
+-   Follow C++'s compilation and linking model
+-   Handling all C++ code
```

Comment:
I don't know what it means by "handle" here. I assume this is referencing the fact that *some* C++ constructs may not have direct interop support between C++ and Carbon? I worry that here, it'll be much more broadly interpreted which I think wouldn't be accurate and might be concerning to readers.